### PR TITLE
:bug: DOP-4185 updates the icons to render properly

### PR DESCRIPTION
**Pull request by caesarbell**
# Pull Request Process

Please read the
[Style Guide Review Process](https://wiki.corp.mongodb.com/display/DE/Technical+Writer+Style+Guide+Review+Process)
wiki page.

Contributors should take the following actions:

- Request a PR review in #docs-style-guide Slack channel.
- Add the Style Guide team reps as reviewers.
- If any reviewer requests changes, address them and request another review.
- When you receive PR review approvals from all the Style Guide team reps,
  they will merge the PR and notify you of the merge in the Slack channel.

# Pull Request Description

Fixes the non-rendering icon issue.

JIRA URL: [<https://jira.mongodb.org/browse/DOCSP-DOP-4185>](https://jira.mongodb.org/browse/DOP-4185)

Staging URL: https://docs-mongodb-org-stg.s3.us-east-2.amazonaws.com/c58679996a06128466e9e7f89a1776f36d0cf21a/DOP-4185/meta/caesarbell/master/reference/icon/index.html

Production URL: https://www.mongodb.com/docs/meta/reference/icon/

### CODE REVIEW

**Comment by caesarbell** [comment]
I couldn't add the reviewer (assuming because it was forked). Adding the reviewer names in the comment @ianf-mongodb @jeff-allen-mongo

**Comment by caesarbell** [comment]
@ianf-mongodb  when you say build log are you referring to the build logs from `snooty-parser`? Sorry if this is a silly question.

**Comment by ianf-mongodb** [comment]
@caesarbell Yeah, typically from the snooty slack bot:

Screenshot 2023-12-12 at 2 21 38 PM


Example:
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=657881c97a5cb5fef55bc16b

**Comment by caesarbell** [comment]
@ianf-mongodb gotcha. Correct me if I am wrong this only happens when the site builds for staging or production correct? What I provided was a staged comment.

File: source/style-guide/includes/rendered-icon-table.rst
Comment (ccho-mongodb) [inline source/style-guide/includes/rendered-icon-table.rst:8856]: Question:
Is this icon invalid? I just wanted to make sure that removing it was intentional.

**Review by ccho-mongodb — APPROVED** [review]
LGTM! Left one question about the last icon.

**Review by jeff-allen-mongo — APPROVED** [review]
LGTM

**Review by ianf-mongodb — APPROVED** [review]
@caesarbell I am not sure of other build methodology or if they provide a log. 

I did take the rst file from this PR: https://raw.githubusercontent.com/mongodb/docs-meta/442a0de01712ecaba48a1f13c16d443e49a089ea/source/style-guide/includes/rendered-icon-table.rst and stage it in the meta docs to [get a build log](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=657b10737a5cb5fef51a36c2).

The only error I got with the update was:
```
ERROR(style-guide/includes/rendered-icon-table.rst:8701ish): The Icon mms-icon-modify does not exist
```

But strangely the icon still renders fine, so I think this LGTM, all other icons previously broken seem to be fixed.

**Comment by ccho-mongodb** [comment]
> @caesarbell I am not sure of other build methodology or if they provide a log.
> 
> I did take the rst file from this PR: https://raw.githubusercontent.com/mongodb/docs-meta/442a0de01712ecaba48a1f13c16d443e49a089ea/source/style-guide/includes/rendered-icon-table.rst and stage it in the meta docs to [get a build log](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=657b10737a5cb5fef51a36c2).
> 
> The only error I got with the update was:
> 
> ```
> ERROR(style-guide/includes/rendered-icon-table.rst:8701ish): The Icon mms-icon-modify does not exist
> ```
> 
> But strangely the icon still renders fine, so I think this LGTM, all other icons previously broken seem to be fixed.

Hi @caesarbell , could you remove the mms-icon-modify for now so that we can avoid having to deal with build errors for all future builds? To address the issue that Ian identified, could you report it to the DOP project in JIRA?

Also, to set up the autobuilder on your for, please follow [this wiki page](https://wiki.corp.mongodb.com/display/DE/How-To%3A+Use+Snooty%27s+Autobuilder+to+Build+Your+Content). If that doesn't work, please check in with the DOP team in Slack.

Thanks!

File: source/style-guide/includes/rendered-icon-table.rst
Comment (ccho-mongodb) [reply source/style-guide/includes/rendered-icon-table.rst:8856]: I think removing is good if it doesn't exist. I just wanted to confirm that it wasn't an accidental deletion.

**Comment by caesarbell** [comment]
@ccho-mongodb will do. I will make those changes and provide the log as well.

**Comment by caesarbell** [comment]
@ccho-mongodb 
@ianf-mongodb 

[Jira](https://jira.mongodb.org/browse/DOP-4212)
[Workpool Build Logs ](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=657c8d257a5cb5fef583a55b)

**Comment by ccho-mongodb** [comment]
> Workpool Build Logs

LGTM!
Will wait for Ian's LGTM to merge.

**Comment by ianf-mongodb** [comment]
One last thing @caesarbell, could you add widths on table? There is a bit of horizontal scroll required to see the entire icon in the table:

Screenshot 2023-12-15 at 4 24 43 PM

**Comment by caesarbell** [comment]
@ianf-mongodb I would argue this is outside of the scope. Could we move that into a separate ticket, as this will also probably be related to Snooty and how it renders the table, thoughts?

**Comment by ccho-mongodb** [comment]
> @ianf-mongodb I would argue this is outside of the scope. Could we move that into a separate ticket, as this will also probably be related to Snooty and how it renders the table, thoughts?

I think changing widths won't help in this case since the length of the fixed width text currently in the first two columns is forcing the width of the third. I don't think there's a great workaround for this styling constraint, so I would suggest skipping addressing and ticketing for now.

**Pull request merged**
